### DISCUSSION TITLE
Feature Request: Save Prefixed IDs Directly to Database

### DISCUSSION THREAD

**Discussion by jony1993** [excid3/prefixed_ids#68] [Ideas]
For our use case, it would be highly beneficial to have an option to save the prefixed IDs directly in the database. This enhancement would offer several advantages:

- Ease of Verification: It simplifies direct database queries when users provide a prefixed ID.
- Data Mapping: Facilitates seamless data mapping between different systems. For example, if our database needs to interface with an external system where the prefixed ID is used as the primary key, this feature would make direct joins and integrations more straightforward.

Is there any easy way to save the prefixed_ids in the database?

**Comment by excid3** [comment]
You can generate tokens for your records in ActiveRecord so I think we would consider that an anti feature for this gem.
